code/codegen/src/main/kotlin/com/android/gradle/replicator/resgen/util/ResgenConstants.kt

```diff
@@ -0,0 +1,18 @@
+package com.android.gradle.replicator.resgen.util
+
+// Vector image
+const val MAX_VECTOR_IMAGE_SIZE_SMALL = 100
```

Comment:
this is not what I meant, I meant to use a configuration file (external file that is loaded by the project replicator). You can however do this in a subsequent CL.

Reply:
Ah, I see. I will do a quick pull request with this then.